spec/Listener/CodeCoverageListenerSpec.php

```diff
@@ -5,15 +5,15 @@
 namespace spec\FriendsOfPhpSpec\PhpSpec\CodeCoverage\Listener;
 
 use FriendsOfPhpSpec\PhpSpec\CodeCoverage\Listener\CodeCoverageListener;
-use PhpSpec\Console\ConsoleIO;
-use PhpSpec\Event\SuiteEvent;
 use PhpSpec\ObjectBehavior;
-use Prophecy\Argument;
-use SebastianBergmann\CodeCoverage\CodeCoverage;
-use SebastianBergmann\CodeCoverage\Filter;
-use SebastianBergmann\CodeCoverage\Report;
 
 /**
+ * Disabled due to tests breaking as php-code-coverage marked their classes
+ * final and we cannot mock them. The tests should be converted into proper
+ * functional (integration) tests instead. This file is left for reference.
+ *
+ * @see https://github.com/leanphp/phpspec-code-coverage/issues/19
```

Comment:
How about adding an interface to those final classes? This would fix all the issues.

Reply:
Adding the interface is not going to be an option, see the comment here: https://github.com/sebastianbergmann/php-code-coverage/pull/833

Reply:
I would prefer to add this in separate commit.

Reply:
I think we actually should not do that. We most probably should stub the Driver, just like the PHPUnit maintainer explained in its comment.